docs/reference/commandline/exec.md

```diff
@@ -39,6 +39,12 @@ The command started using `docker exec` only runs while the container's primary
 process (`PID 1`) is running, and it is not restarted if the container is
 restarted.
 
+> **Note**: COMMAND run using the last `WORKDIR` specified.
```

Comment:
To understand this, the reader has to know that `WORKDIR` is a Dockerfile directive.

How about something like: "COMMAND will run in the default directory of the container. For UNIX systems, this is `/` and for Windows [I got no idea]. It the underlying image has a custom directory specified with the `WORKDIR` directive in its Dockerfile, this will be used instead."